### PR TITLE
Import color scheme from Figma

### PR DESCRIPTION
**Pull request by Phon1209**
After merging this PR, the color will mostly stop working, since the naming convention has changed.

### CODE REVIEW

**Comment by martig7** [comment]
I'm going to just implement all of the CSS on this branch before merging so that I can compare with master.

**Comment by martig7** [comment]
Okay this should be ready for merging now.

**Comment by martig7** [comment]
@Phon1209 Let me know if I did anything wrong.

**Comment by Phon1209** [comment]
There's a color specific for certain uses:
text: text-primary text-secondary text-tertiary quaternary 
border: border-border-primary and so on

I wasn't clear when we have a meeting, but you can inspect what color you should use in figma.

**Comment by Phon1209** [comment]
image

You can check them out in the tab on the right

**Comment by martig7** [comment]
Ah. I'll fix that.

**Comment by martig7** [comment]
Additionally Closes #53

File: app/components/navigation/NavButton.tsx [hunk 1]
```diff
@@ -22,8 +22,8 @@ const NavButton = ({
   return (
     <Link href={link}>
       <div
-        className={`flex items-center py-2 px-3 gap-2 rounded-md hover:bg-gray-50 ${
-          selected && "text-primary-800 bg-gray-100"
+        className={`flex items-center py-2 px-3 gap-2 rounded-md hover:bg-utility-gray-300 ${
+          selected && "text-utility-brand-800 bg-utility-gray-100"
         }`}
```
Comment (Phon1209) [inline app/components/navigation/NavButton.tsx:27]: use btn class

File: app/components/pathway/FilterComponent.tsx [hunk 2]
```diff
@@ -66,7 +66,7 @@ export const ModeRadioButton = ({
   return (
     <button
       className={`checkbox-group !border-solid ${
-        checked ? "checked !bg-primary-50" : ""
+        checked ? "checked !bg-bg-brand-secondary" : ""
       }`}
```
Comment (Phon1209) [inline app/components/pathway/FilterComponent.tsx:70]: cn please

File: app/components.css [hunk 3]
```diff
@@ -0,0 +1,167 @@
...
+
+  .status-bar .status.status__in-progress {
+    @apply bg-bg-warning-secondary;
+  }
+
+  .status-bar .status.status__planned,
+  .status-bar .status.status__not-started {
+    @apply bg-fg-disabled_subtle;
+  }
+}
```
Comment (Phon1209) [inline app/components.css:167]: I have made a custom class for most stuff already. Please use this from now on, only create new one when cannot find here.

File: app/typography.css [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+    @apply text-display-sm;
+    @apply xl:text-display-md;
+  }
+
+  .texta-display-md {
+    @apply md:text-display-sm;
+    @apply text-display-md;
+    @apply xl:text-display-lg;
+  }
+}
```
Comment (Phon1209) [inline app/typography.css:46]: Same goes for the text size class.  while it's not mandatory to do this right now, it's better to start using it to enforce the same standard.

btw, feel free to change the text classname from `texta` to something else if you think it doesn't making sense

Comment (martig7) [reply app/typography.css:46]: alright I'll keep both in mind

**Comment by Phon1209** [comment]
You could also add more utility class if you think it's appropriate as well; don't just rely on the existing one.

**Review by EvWhymark — APPROVED** [review]
LGTM

**Comment by martig7** [comment]
I'll fix the classes in another PR, I just wanted to get this one merged ASAP so that I can start on other issues.

**Pull request merged**